### ISSUE TITLE
Evaluate model (i.e. get loss of test set w.r.t to trained model)

### ISSUE THREAD

**Issue by pltrdy** [facebookresearch/fastText#677]
Hi,

I've been looking for a way to calculate loss on a kind of test set on a trained model.

The idea is to evaluate if adding general data is better for my model or not i.e. I have two training dataset `A` and `B` and a test-set `T`, I would like to train a model based on `A`, and on  `A+B` and see if this data augmentation improves the model.

**Comment by Celebio** [comment]
Hi @pltrdy ,
We are planning to add the loss as a result of the `test` command soon.

Regards,
Onur

**Comment by pltrdy** [comment]
It seems that `test` is related to classification, right?

I'm talking about the embeddings only, i.e. fastTest training provides the training loss, I'd like to have a validation/test loss as well. Is it what you're talking about?

**Comment by iedmrc** [comment]
Any update on this?
